build.gradle

```diff
@@ -60,6 +71,11 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
     from javadoc.destinationDir
 }
 
+jar {
+    // add jakarta classier for Jakarta EE 9
+    archiveClassifier = isJakarta ? "jakarta" : ""
```

Comment:
is emtpy string the default value, or should it be null?

Reply:
I am not sure the default value of the *classifier*, I think empty string works.